### ISSUE TITLE
Content-Disposition header not persisted on blob upload

### ISSUE THREAD

**Issue by jarroda** [Azure/azure-storage-net#353]
# When uploading a new blob, the `Content-Disposition` header does not persist.  Setting Content-Type, Metadata etc seems to persist when uploading a new blob, but the only way the `Content-Disposition` header persists is with a call to `SetProperties`.

I am uploading a new blob like so:

```
var blob = container.GetBlockBlobReference(blobName);
blob.Properties.ContentType = someContentType;                
blob.Properties.ContentDisposition = someContentDisposition;

await blob.UploadFromStreamAsync(myStream);
await blob.FetchAttributesAsync();

Assert.True(blob.Properties.ContentType == someContentType); // Passes
Assert.True(blob.Properties.ContentDisposition == someContentDisposition); // Fails
```

**Comment by erezvani1529** [comment]
Hi @jarroda,

Would you please share the Storage Client package version and the project type you are working with?

Thanks!

**Comment by erezvani1529** [comment]
Hi @jarroda,

Thanks for bringing this issue to our attention. We have a fix for it which will be rolled out with our 8.0 release(we will update this thread with the respective PR).

Thanks!

**Comment by guilmori** [comment]
Looking for this fix as well.

I'm having the same issue using this code to upload from javascript:
https://github.com/kinstephen/angular-azure-blob-upload

If I set Content-Disposition header, it is not saved into the blob.

**Comment by erezvani1529** [comment]
Hi @jarroda , @guilmori ,

Closing this issue as the fix is included in the latest published package on nuget [v8.0.0](https://www.nuget.org/packages/WindowsAzure.Storage/) 

Thanks!
